components/webhook/Form.tsx

```diff
@@ -26,7 +26,7 @@ const Form = ({
     validationSchema: Yup.object().shape({
       name: Yup.string().required(),
       url: Yup.string().required().url(),
-      eventTypes: Yup.array(),
+      eventTypes: Yup.array().required().min(1),
```

Comment:
@jatingodnani Please update the error message with `Please choose at least one event type.`

Reply:
It is already present you can check

Reply:
At the moment the error message says `eventTypes field must have at least 1 items`. Please check it and replace it.

Reply:
Ok